Marine law and SOP update

QuietlyWhisper opened the pull request:
Pls merge with 

- #8445

**Changelog**
<!-- Add a Changelog entry to make players aware of new features or changes that could affect gameplay.
Admin changes may be listed for admins to see with admin:
Coding changes with no changes visible in-game may be listed for other contributors with code:
Make sure to read the guidelines.
Changelog must have a :cl: symbol, so the bot recognizes the changes and adds them to the game's changelog. -->
:cl: Whisper
- remove: (General Change) Officer Ordered Arrests and all their related mentions and provisions have been removed - excluding those given to Commanding Officers.
- add: (Marine Law) Added text explaining the goals of the MP and an example of problem-solving.
- add: (Law, Definitions) Added a definition of High Ranking Officers and a link to their provisions.
- add: (Law, Definitions) Standard Issue Equipment definition added, equipment SOP made more clear.
- fix: (Law, Arresting) More leeway is granted for accidental bodily harm while deployed.
- tweak: (Law, Arresting) Field Arrests have been reworked. This includes a loosening of movement restrictions of groundside MPs, when given approval.
- add: (Law, Arresting) MP may warn people for minor crimes.
- add: (Law, Arresting) Added Investigative detainment, and time restrictions for processing and completing arrests.
- tweak: (Law, Charging) This page has been reorganized for ease of review - following issues of not following procedure.
- add: (Prisoner Rights)  Added prisoner rights to have clothing.
- add: (Law, Synthetics) Synthetics not part of either the UNMC or WeYA now have rights.
- fix: (Law, Variable Crimes) Insubordination cannot be applied to orders that are made with the intent to arrest - this should be resisting arrest.
- tweak: (Law, Variable Crimes) Optional crimes have been merged into variable crimes.
- add: (Law, Variable Crimes) Public Mischief has been added as a variable crime. It relates to false statement that accuses someone of having committed an offence, and similar behavior of wasting time.
- tweak: (Law, Variable Crimes) Cleaned up the text of the Resisting Arrest crime.
- add: (SOP, Equipment) Added regulations for the command tablet.
- add: (Rules, Core 5) Added a notice that if behavior in the Military Police role does not meet community standards, rolebans may be applied less leniently. This may not be an indication of a bad player, but it is an indication that a player doesn't have the understanding required to play MP.
- tweak: (Rules, Command) Command rules have been condensed into a single page.
- add: (Rules, Command) MP to the best of their ability should be investigating reports of crimes.
- add: (Marine Roleplay Standards) Do not use setpose for anything but setting an examinable character pose - such as leaning on a wall.
- tweak: (Law, Minor Crimes) Remade DASO into Disrespecting a Superior, which is an NJP charge that applies to any higher ranking person.
- fix: Minor guidebook spelling and consistency fixes by CatAndHats

QuietlyWhisper committed:
Marine law and SOP update

QuietlyWhisper committed:
fix some weird text in crimes

QuietlyWhisper committed:
more leeway for bodily harm groundside

QuietlyWhisper committed:
add a link to higher ranking officers in definitions

QuietlyWhisper committed:
remove OOA reference from njp info

QuietlyWhisper committed:
rework field arrests, remove implied ooa

QuietlyWhisper committed:
fix an issue with survivor synths

QuietlyWhisper committed:
modify insubordination to exclude orders related to arrests

QuietlyWhisper committed:
Combine optional crimes with variable crimes.

QuietlyWhisper committed:
add Public Mischief crime

QuietlyWhisper committed:
suggestion, not have to

QuietlyWhisper committed:
bold usage of "AO"

QuietlyWhisper committed:
update wording of resisting arrest

QuietlyWhisper committed:
further warnings for mp behavior

QuietlyWhisper committed:
Add a spirit of the law example, remove ooa, add investigative detainment and time restrictions for arrests

PursuitInAshes commented on Resources/ServerInfo/Guidebook/_RMC14/Law/RMCMarineLawArresting.xml:
```diff
@@ -4,20 +4,23 @@
+  Unless an investigation becomes an arrest, a suspect being investigated should not be moved away from their duties, such as the frontline, but should still be kept relatively safe.
+  - [color=#ff000d][bold]MP[/bold][/color] may detain and physically restrain a suspect, with reasonable suspicion that they are involved in a crime, for up to 10 minutes to gather evidence and witness testimony.
+  - This time used also carries over to an arrest, which will not grant an extra 10 minutes.
+  - If these 10 minutes pass during the investigation, the suspect is to be immediately released if not found guilty of any crimes. If found guilty of crimes, they must still not be detained longer than 10 minutes, and be processed into arresting and charging procedure with reasonable speed.
```
"If these 10 minutes pass during the investigation, the suspect is to be immediately released if not found guilty of any crimes. If found guilty of crimes, they must still not be detained longer than 10 minutes, and **instead** be processed into arresting and charging procedure with reasonable speed."

(For some reason github removed my nice "Suggest Changes" button where you could just commit the change if you wanted it, sorry about that)

PursuitInAshes reviewed (commented):
Both of these are minor and I don't know if they match the intent/focus you have, but figured I'd point them out. If they don't match/you feel they aren't need feel free to just ignore them, no worries

QuietlyWhisper committed:
missed a spot

QuietlyWhisper committed:
throw another line on there just for further clarity

Timemaster99 commented on Resources/ServerInfo/Guidebook/_RMC14/Law/RMCMarineLaw.xml:
```diff
@@ -5,6 +5,8 @@
 
   The goal of [color=#ff000d][bold]MP[/bold][/color] is to make the ship safer and productive place for everyone. Do your best to uphold that goal, by mediating conflicts and assisting the crew and marines with their needs, using the minimal force required.
 
+  Always consider first if an arrest is actually needed to resolve a conflict. Arresting a doctor immediately for [bold]Neglect of Duty[/bold] will not make medical duties be performed more efficiently since they would be in the brig - but if they actually refuse to do their duties, then it would make sense to perform an arrest.
```
This should remain to define the overarching intent of the text, but I believe we should accompany it by changing the classifications of some crimes to directly avoid charging it in the instances outlined above.

Neglect of Duty, for example, can be put into Variable Crimes, where 
**Minor**: _Unintentional failure to perform one's role to an acceptable standard_ | Correction of behavior
**Major**: _Intentional refusal to perform one's role to an acceptable standard_ | 15 minutes (unchanged)

This sets up a mandatory warning where unintentional mistakes are corrected, and only punished if the behavior continues.

Timemaster99 commented on Resources/ServerInfo/Guidebook/_RMC14/Law/RMCMarineLawArresting.xml:
```diff
@@ -4,20 +4,23 @@
+  - This time used also carries over to an arrest, which will not grant an extra 10 minutes.
+  - If these 10 minutes pass during the investigation, the suspect is to be immediately released if not found guilty of any crimes. If found guilty of crimes, they must still not be detained longer than 10 minutes, and be processed into arresting and charging procedure with reasonable speed.
+  - Should the suspect be found innocent of all accused crimes, the suspect should be released, and all investigations being done by [color=#ff000d][bold]MP[/bold][/color] ended.
+  - The [color=#8B0000][bold]Chief MP[/bold][/color] may order the release of a suspect under Investigative Detainment.
```
```suggestion
  - The [color=#8B0000][bold]Chief MP[/bold][/color] may order the early release of a suspect under Investigative Detainment at their discretion. This does not prevent the suspect from being charged if found guilty before or after this order.
```
Assuming the intention here is to allow CMPs to exercise judgement to end Investigative Detainments without having to fulfill the requirements of "gather[ing] evidence and witness testimony." We specify this so that the detaining MPs don't challenge the order.

QuietlyWhisper committed:
capitalize marine law

QuietlyWhisper committed:
capitalize marine law

QuietlyWhisper committed:
Law proofreading

QuietlyWhisper committed:
Combine all three command rules pages into one page.

Added Executive Officers are expected to have a reasonable understanding of Marine Law and SOP

QuietlyWhisper committed:
proofread

QuietlyWhisper committed:
remove more OOA references

QuietlyWhisper committed:
My IDE probably broke the formatting here

QuietlyWhisper committed:
remove a mention of OOA

QuietlyWhisper committed:
Add to ignoring minor crimes - may warn, and njp (though njps were always allowed)

QuietlyWhisper committed:
standard issue equipment moved to definition. Add SOP for the command tablet, so it is not stolen. Change nonmod department to explicitly mention equipment as well.

QuietlyWhisper committed:
remove duplicated fob text

QuietlyWhisper committed:
cryo > Hypersleep

QuietlyWhisper committed:
fix mp tacmap icons

QuietlyWhisper committed:
Reorganize charging page

QuietlyWhisper committed:
Recent changes

QuietlyWhisper committed:
typo

QuietlyWhisper committed:
organization and a typo

QuietlyWhisper committed:
added clothing rights, added rule for setpose

PursuitInAshes reviewed (approved):
'Code' lgtm, no phrasing changes I'd suggest either.
Said it before and I'll say it again, glad these are coming.

QuietlyWhisper committed:
Fix a carryover issue from ss14 - no changelog needed

QuietlyWhisper committed:
Separate insubordination into disrespect and following orders. DASO is removed and disrespect is NJP only.

QuietlyWhisper committed:
add a character name guideline
